packages/backend-function/src/function_env_type_generator.test.ts

```diff
@@ -12,6 +12,7 @@ void describe('FunctionEnvironmentTypeGenerator', () => {
     await fsp.rm(path.join(process.cwd(), '.amplify'), {
       recursive: true,
       force: true,
+      maxRetries: 3,
```

Comment:
TIL that `fs` api has this.

Reply:
Does this only exist on `rm`?

Reply:
`rm`, `rmDir` and their synchronous counterparts.